src/guides/protocol.md

```diff
@@ -0,0 +1,271 @@
+## Handshake Protocol Summary
+
+This guide is intended for developers with a working knowledge of the Bitcoin
+protocol and its general consensus parameters. All values below describe mainnet
+parameters.
+[Testnet, regtest, and simnet may differ.](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/protocol/networks.js)
+Links reference the `hsd` codebase at the time of writing, during Testnet 4.
+
+### Blockchain
+
+- Block interval time:
+[10 minutes](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/protocol/networks.js#L166)
+
+- PoW Difficulty adjustment interval: [144 Blocks (one day)](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/protocol/networks.js#L158-L159)
+
+- Maximum block size: [1000000 Bytes base block size](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/protocol/consensus.js#L160),
+[4000000 Weight Units](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/protocol/consensus.js#L176-L177)
+
+- Hash algorithm: "cBLAKE" --
+[BLAKE2b256 (sha3) keyed with KMAC256](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/mining/mine.js#L28-L29)
```

Comment:
BLAKE2b256 is not sha3, the sha3 part comes from KMAC256

Reply:
I see, thanks - so maybe better phrasing is `BLAKE2b256 keyed with KMAC256 (sha3)` ?

Reply:
That sounds good